R/tostring.R

```diff
@@ -666,6 +667,12 @@ setMethod("toString", "MatrixPrintForm", function(x,
   footer_txt
 }
 
+reorder_ref_fnotes <- function(fns) {
+  ind <- gsub("\\{(.*)\\}.*", "\\1", fns)
+  ind_num <- suppressWarnings(as.numeric(ind))
+  if (!any(is.na(ind_num))) fns[order(ind_num)] else fns
```

Comment:
it is na when there are * right? If that is the case do we want to order them anyway? Like putting them first ordered as it was and then the others?